addons/web/static/tests/core/confirmation_dialog_tests.js

```diff
@@ -44,21 +44,52 @@ QUnit.module("Components", (hooks) => {
                 close: () => {
                     assert.step("Close action");
                 },
-                confirm: () => {},
+                confirm: () => {
+                    throw new Error("should not be called");
+                },
                 cancel: () => {
-                    assert.step("Cancel action");
+                    throw new Error("should not be called");
+                },
+                dismiss: () => {
+                    assert.step("Dismiss action");
                 },
             },
         });
         assert.verifySteps([]);
         triggerHotkey("escape");
         await nextTick();
         assert.verifySteps(
-            ["Cancel action", "Close action"],
+            ["Dismiss action", "Close action"],
             "dialog has called its cancel method before its closure"
```

Comment:
Should be modified to:

"dialog has called its dismiss method before its closure"